lib/bamboo/strategies/deliver_immediately_strategy.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Bamboo.DeliverImmediatelyStrategy do
```

Comment:
I’m not sure why, but I like ImmediateDeliveryStrategy a lot more. That’s a great suggestion!

> On Feb 17, 2016, at 12:23 PM, Steve Domin <notifications@github.com> wrote:
> 
> In lib/bamboo/strategies/deliver_immediately_strategy.ex <https://github.com/paulcsmith/bamboo/pull/71#discussion_r53198773>:
> 
> > @@ -0,0 +1,13 @@
> > +defmodule Bamboo.DeliverImmediatelyStrategy do
> Minor point, what do you think of ImmediateDeliveryStrategy as a module name?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/paulcsmith/bamboo/pull/71/files#r53198773>.
>